model/minsky.cc

```diff
@@ -266,10 +266,8 @@ namespace minsky
     m.populateGroup(*g);
     // Default pasting no longer occurs as grouped items or as a group within a group. Fix for tickets 1080/1098    
     auto p = model;
-	p->moveContents(*g);
-    for (auto& i: p->items)   
-        canvas.selection.ensureItemInserted(i);  
-    model->removeGroup(*g);
+	p->moveContentsInSelection(*g);
+    p->removeGroup(*g);
```

Comment:
I don't think this line nor 265 is needed.

Reply:
When I removed them, I got a segmentation fault during tests t0026a.sh and t0036a.sh, so I put them back in.

Reply:
What was the cause of the segfault?

Reply:
Does removeGroup also clear itemFocus?

Reply:
> What was the cause of the segfault?

I followed the steps in t0036a.sh in gdb and the following backtrace:

![Screenshot_segfault_ticket1080](https://user-images.githubusercontent.com/46541934/71604528-e959fe80-2b62-11ea-8678-30872564b67c.png)

Reply:
> Does removeGroup also clear itemFocus?

No, it doesn't look like it does to me:

  GroupPtr GroupItems::removeGroup(const Group& group)
  {
    for (auto i=groups.begin(); i!=groups.end(); ++i)
      if (i->get()==&group)
        {
          GroupPtr r=*i;
          groups.erase(i);
          return r;
        }

    for (auto& g: groups)
      if (GroupPtr r=g->removeGroup(group))
        return r;
    return GroupPtr();
  }

Reply:
> What was the cause of the segfault?

I attach another screenshot with more details on frames 5-7 of the backtrace at the bottom:

![Screenshot_segfault_backtrace_frames_ticket1080](https://user-images.githubusercontent.com/46541934/71604688-1c50c200-2b64-11ea-8b6b-8daf96023611.png)